src/mame/vectorgraphic/vector4.cpp

```diff
@@ -103,6 +108,8 @@ class vector4_state : public driver_device
 	memory_view m_romenbl;
 	required_device<vector_sbc_video_device> m_sbc_video;
 	required_device<s100_bus_device> m_s100;
+	required_device<centronics_device> m_centronics;
+	uint8_t m_ppi_pc;
```

Comment:
Thank you for the review and noticing these basic mistakes. I'm annoyed missing initialization kept happening...

I am now initializing it within the constructor. I can't initialize from m_centronics because centronics_device only has the callbacks. output_latch_device is the only state.